### DISCUSSION TITLE
Multilanguage

### DISCUSSION THREAD

**Discussion by rkaldung** [tobimori/kirby-dreamform#72] [General] [answered]
Hello,

I have a multilanguage site and added a form where the form labels are translated in the panel. The rendered form always shows the labels of the default language. Did I miss something?

Thank you

**Comment by tobimori** [comment]
This should definitely not be the case, however I can't reproduce this issue on my setup. Can you send the support an email with a backup of the site so we can take a look?

**Reply by rkaldung** [reply, marked as answer]
Found the issue, ISO layer 8. Sorry for the noise and thank you for that great plugin ❤️
